api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -57,6 +57,16 @@ message RedisProxy {
       ANY = 4;
     }
 
+    // Protocol controls how Envoy parse client's request. This is currently
+    // supported protocols are memcached and redis.
+    enum Protocol {
+      // Default mode. Parse for redis.
+      Redis = 0 [(udpa.annotations.enum_value_migrate).rename = "REDIS"];
+
+      // Parse for memcached.
```

Comment:
Can you link to how Redis treats memcached requests? Some documentation would help.

Reply:
Otherwise LGTM for API.